repository/i.json

```diff
@@ -810,6 +810,17 @@
 			]
 		},
 		{
+			"name": "Ionic 2 Package",
```

Comment:
Hi @wbond ,

I will improve this package with more things. Like completions. Because this I put this name with ionic 2 package.